index.html

```diff
@@ -73,7 +73,298 @@ <h1>I'm a software developer</h1>
         </ul>
       </div>
       <!-- End of social media -->
+      <div class="button">
+        <a href="#">
+          <img src="images/scroll-button.png" alt="scroll-button" />
+        </a>
+      </div>
+    </section>
+    <!-- *start of work section -->
+    <section class="works">
+      <!-- h1, a line, 6 x cards, 
+        resume, languages, frameworks, skills  -->
+      <div>
+        <h1>My Recent Works</h1>
+      </div>
+      <div class="line"></div>
+      <!-- start of cards -->
+      <!-- <div> -->
+      <!-- h2, 4 x buttons, see-project button -->
+      <div class="card-1">
+        <div class="h2">
+          <h2>
+            Multi-Post Stories<br />
+            Gain+Glory
+          </h2>
+        </div>
+        <div class="card-buttons">
+          <a href="#"
+            ><img class="button1" src="images/ror.png" alt="ror-button"
+          /></a>
+          <a href="#"
+            ><img class="button2" src="images/css.png" alt="css-button"
+          /></a>
+          <a href="#"
+            ><img class="button3" src="images/jvscr.png" alt="javascr-button"
+          /></a>
+          <a href="#"
+            ><img class="button4" src="images/html.png" alt="html-button"
+          /></a>
+        </div>
```

Comment:
- [ ] Good job setting all the elements here :+1: but as the project requires it should be a list, so you should use `<li>` tags for each `<a>` element and wrap all of them using `<ul>` :+1: